### ISSUE TITLE
uncompr error with content()

### ISSUE THREAD

**Issue by rob-2018** [pmqs/Archive-Zip-SimpleZip#4]
`$data=$z->content("foo.txt");` gives error: `Can't call method "uncompr" on an undefined value at /usr/share/perl5/IO/Uncompress/Base.pm line 917.`

This happens on different machines with slightly different versions of used modules, eg IO::Compress 2.081 and 2.084 so it looks like the error is caused by simpleunzip.

**Comment by pmqs** [comment]
Issue reproduced

```
# echo abcd | zip /tmp/x.zip -
  adding: - (stored 0%)
# perl -MArchive::Zip::SimpleUnzip -e 'print Archive::Zip::SimpleUnzip->new( "/tmp/x.zip")->content("-")'
Can't call method "uncompr" on an undefined value at /usr/local/share/perl5/IO/Uncompress/Base.pm line 915.
```

Not an issue with the `content` method from the member.

```
# echo abcd | zip /tmp/x.zip -
  adding: - (stored 0%)
# perl -MArchive::Zip::SimpleUnzip -e 'print Archive::Zip::SimpleUnzip->new("/tmp/x.zip")->next()->content()'
abcd
```

**Comment by pmqs** [comment]
Fixed in efede5832f33b403c767a458c22fbc7d1f21c672
